enhancements/network/service-egress-traffic-steering.md

```diff
@@ -86,11 +93,11 @@ $ ovn-nbctl lr-policy-list ovn_cluster_router
        101      ip4.src == 10.244.1.6/32                                       reroute      10.244.0.2
        ...
 ```
-After that the service will be annotated with `k8s.ovn.org/egress-service-host=<node_name>` and the node labeled with `egress-service.k8s.ovn.org/<svc-namespace>-<svc-name>: ""`.
+After that the `EgressService` resource status field will be updated with `host: <node_name>` and the node labeled with `egress-service.k8s.ovn.org/<svc-namespace>-<svc-name>: ""`.
```

Comment:
if you used the 1to1 name mapping im not sure why you would need to include the name of the service in the label, but if you go the other route then it makes sense. For the 1 to many mapping, \maybe it makes more sense to have an AppliedTo field and a list of all services where EgressService has been applied.

Reply:
the label is meant for the LoadBalancer provider (like MetalLB) to select the labeled node for that service, so this behavior doesn't change between the old annotation and what's proposed here

Reply:
ah ok